nvflare/lighter/poc_commands.py

```diff
@@ -209,6 +209,9 @@ def stop_poc(poc_workspace: str, excluded=None, white_list=None):
 
         print("shutdown NVFLARE")
         sess.api.do_command("shutdown all")
+
+        wait_for_system_shutdown(sess)
+
```

Comment:
Why if this session wait for system to shutdown will have less error messages?
This session should have nothing to do with the messages on either server or client side?